.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,87 @@
+name: Bug report
+description: Create a bug report to help us improve
+labels:
+  - bug
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of the bug.
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: OpenWrt version
+      description: The OpenWrt release or commit hash where this bug occurs (use command below).
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ```. /etc/openwrt_release && echo $DISTRIB_REVISION```
+  - type: input
+    id: target
+    attributes:
+      label: OpenWrt target/subtarget
+      description: The OpenWrt target and subtarget where this bug is observed (use command below).
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ```. /etc/openwrt/release && echo $DISTRIB_TARGET```
+  - type: input
+    id: device
+    attributes:
+      label: Device
+      description: The device exhibiting this bug.
+    validations:
+      required: true
+  - type: dropdown
+    id: image_kind
+    attributes:
+      label: Image kind
+      options:
+        - Official downloaded image
+        - Self-built image
+    validations:
+      required: true
+  - type: textarea
+    id: reproduce
+    attributes:
+      label: Steps to reproduce
+      description: Steps to reproduce the reported behaviour.
+  - type: textarea
+    id: behaviour
+    attributes:
+      label: Actual behaviour
+      description: A clear and concise description of what actually happens.
+  - type: textarea
+    id: expected
+    attributes:
+      label: Expected behaviour
+      description: A clear and concise description of what you expected to happen.
+  - type: textarea
+    id: additional
+    attributes:
+      label: Additional info
+      description: Add any additional info you think might be helfpul.
+  - type: textarea
+    id: diffconfig
+    attributes:
+      label: Diffconfig
+      description: In case of a self-built image, please attach diffconfig.
```

For any "code" textarea, it is easier to read if you add a render attribute. It will add the three batticks to the text included by the user.
```suggestion
  - type: textarea
    id: diffconfig
    attributes:
      label: Diffconfig
      description: In case of a self-built image, please attach diffconfig.
    render: text
```